pages/common/tree.md

```diff
@@ -14,14 +14,14 @@
 
 `tree -a`
 
-- Print human readable size of files:
+- Print the tree without indentation lines, showing the full path instead:
 
-`tree -h`
+`tree -i -f`
 
-- Print the full path for each file:
+- Print the size of each node next to it, in human-readable format, with folders displaying their cumulative size (as in the `du` command):
```

Comment:
Its not the length of the description. Although a shorter description does have lesser cognitive load.

But yea .. I agree the 3 options really need to go together. In that case, your revised statement certainly looks better.